### ISSUE TITLE
The viewer does not appear after running run_main.py

### ISSUE THREAD

**Issue by fisherrrr** [Florian-Barthel/splatviz#33]
After installing the gs-view environment, the viewer does not appear after running run_main.py.

Win10 

>packages in environment at C:\Users\PC16\anaconda3\envs\gs-view:
annotated-types           0.7.0              pyhd8ed1ab_0    conda-forge
aom                       3.9.1                he0c23c2_0    conda-forge
blas                      2.112                       mkl    conda-forge
blas-devel                3.9.0              12_win64_mkl    conda-forge
brotli-python             1.1.0           py310h00ffb61_1    conda-forge
bzip2                     1.0.8                h2466b09_7    conda-forge
ca-certificates           2023.08.22           haa95532_0    anaconda
cairo                     1.18.0               h91e5215_2    conda-forge
certifi                   2023.11.17      py310haa95532_0    anaconda
cffi                      1.16.0          py310h8d17308_0    conda-forge
charset-normalizer        3.3.2              pyhd8ed1ab_0    conda-forge
click                     8.1.7           win_pyh7428d3b_0    conda-forge
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
contourpy                 1.2.1                    pypi_0    pypi
cuda-cccl                 12.4.127                      0    nvidia
cuda-cudart               11.8.89                       0    nvidia
cuda-cudart-dev           11.8.89                       0    nvidia
cuda-cupti                11.8.87                       0    nvidia
cuda-libraries            11.8.0                        0    nvidia
cuda-libraries-dev        11.8.0                        0    nvidia
cuda-nvrtc                11.8.89                       0    nvidia
cuda-nvrtc-dev            11.8.89                       0    nvidia
cuda-nvtx                 11.8.86                       0    nvidia
cuda-profiler-api         12.4.127                      0    nvidia
cuda-runtime              11.8.0                        0    nvidia
cudatoolkit               11.8.0              h09e9e62_13    conda-forge
cycler                    0.12.1                   pypi_0    pypi
dav1d                     1.2.1                hcfcfb64_0    conda-forge
diff-gauss                1.0.5                    pypi_0    pypi
double-conversion         3.3.0                h63175ca_0    conda-forge
expat                     2.6.2                h63175ca_0    conda-forge
ffmpeg                    6.1.1           gpl_h52c3a39_114    conda-forge
filelock                  3.15.4             pyhd8ed1ab_0    conda-forge
font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
font-ttf-ubuntu           0.83                 h77eed37_2    conda-forge
fontconfig                2.14.2               hbde0cde_0    conda-forge
fonts-conda-ecosystem     1                             0    conda-forge
fonts-conda-forge         1                             0    conda-forge
fonttools                 4.53.1                   pypi_0    pypi
freeglut                  3.2.2                he0c23c2_3    conda-forge
freetype                  2.12.1               hdaf720e_2    conda-forge
fsspec                    2024.6.1                 pypi_0    pypi
glfw                      2.7.0                    pypi_0    pypi
gputil                    1.4.0                    pypi_0    pypi
graphite2                 1.3.13            h63175ca_1003    conda-forge
h2                        4.1.0              pyhd8ed1ab_0    conda-forge
harfbuzz                  8.5.0                h81778c3_0    conda-forge
hdf5                      1.14.3          nompi_h2b43c12_105    conda-forge
hpack                     4.0.0              pyh9f0ad1d_0    conda-forge
hyperframe                6.0.1              pyhd8ed1ab_0    conda-forge
icu                       73.2                 h63175ca_0    conda-forge
idna                      3.7                pyhd8ed1ab_0    conda-forge
imagecodecs               2024.6.1                 pypi_0    pypi
imageio                   2.34.2             pyh12aca89_0    conda-forge
imageio-ffmpeg            0.5.1              pyhd8ed1ab_0    conda-forge
imath                     3.1.11               h12be248_0    conda-forge
imgui                     2.0.0                    pypi_0    pypi
imgui-bundle              1.5.2                    pypi_0    pypi
intel-openmp              2021.4.0                 pypi_0    pypi
jasper                    4.2.4                hcb1a123_0    conda-forge
jinja2                    3.1.4              pyhd8ed1ab_0    conda-forge
khronos-opencl-icd-loader 2023.04.17           h64bf75a_1    conda-forge
kiwisolver                1.4.5                    pypi_0    pypi
krb5                      1.21.3               hdf4eb48_0    conda-forge
lazy-loader               0.4                      pypi_0    pypi
lcms2                     2.16                 h67d730c_0    conda-forge
lerc                      4.0.0                h63175ca_0    conda-forge
libabseil                 20240116.2      cxx17_he0c23c2_1    conda-forge
libaec                    1.1.3                h63175ca_0    conda-forge
libasprintf               0.22.5               h5728263_2    conda-forge
libblas                   3.9.0              12_win64_mkl    conda-forge
libcblas                  3.9.0              12_win64_mkl    conda-forge
libclang13                18.1.8          default_ha5278ca_1    conda-forge
libcublas                 11.11.3.6                     0    nvidia
libcublas-dev             11.11.3.6                     0    nvidia
libcufft                  10.9.0.58                     0    nvidia
libcufft-dev              10.9.0.58                     0    nvidia
libcurand                 10.3.5.147                    0    nvidia
libcurand-dev             10.3.5.147                    0    nvidia
libcurl                   8.9.0                h18fefc2_0    conda-forge
libcusolver               11.4.1.48                     0    nvidia
libcusolver-dev           11.4.1.48                     0    nvidia
libcusparse               11.7.5.86                     0    nvidia
libcusparse-dev           11.7.5.86                     0    nvidia
libdeflate                1.20                 hcfcfb64_0    conda-forge
libexpat                  2.6.2                h63175ca_0    conda-forge
libffi                    3.4.2                h8ffe710_5    conda-forge
libgettextpo              0.22.5               h5728263_2    conda-forge
libglib                   2.80.2               h0df6a38_0    conda-forge
libhwloc                  2.11.1          default_h8125262_1000    conda-forge
libiconv                  1.17                 hcfcfb64_2    conda-forge
libintl                   0.22.5               h5728263_2    conda-forge
libjpeg-turbo             3.0.0                hcfcfb64_1    conda-forge
liblapack                 3.9.0              12_win64_mkl    conda-forge
liblapacke                3.9.0              12_win64_mkl    conda-forge
libnpp                    11.8.0.86                     0    nvidia
libnpp-dev                11.8.0.86                     0    nvidia
libnvjpeg                 11.9.0.86                     0    nvidia
libnvjpeg-dev             11.9.0.86                     0    nvidia
libopencv                 4.9.0           qt6_py310h204659d_615    conda-forge
libopenvino               2024.1.0             hfe1841e_7    conda-forge
libopenvino-auto-batch-plugin 2024.1.0             h04f32e0_7    conda-forge
libopenvino-auto-plugin   2024.1.0             h04f32e0_7    conda-forge
libopenvino-hetero-plugin 2024.1.0             h372dad0_7    conda-forge
libopenvino-intel-cpu-plugin 2024.1.0             hfe1841e_7    conda-forge
libopenvino-intel-gpu-plugin 2024.1.0             hfe1841e_7    conda-forge
libopenvino-ir-frontend   2024.1.0             h372dad0_7    conda-forge
libopenvino-onnx-frontend 2024.1.0             hdeef14f_7    conda-forge
libopenvino-paddle-frontend 2024.1.0             hdeef14f_7    conda-forge
libopenvino-pytorch-frontend 2024.1.0             he0c23c2_7    conda-forge
libopenvino-tensorflow-frontend 2024.1.0             h7c40eac_7    conda-forge
libopenvino-tensorflow-lite-frontend 2024.1.0             he0c23c2_7    conda-forge
libopus                   1.3.1                h8ffe710_1    conda-forge
libpng                    1.6.43               h19919ed_0    conda-forge
libprotobuf               4.25.3               h503648d_0    conda-forge
libsqlite                 3.46.0               h2466b09_0    conda-forge
libssh2                   1.11.0               h7dfc565_0    conda-forge
libtiff                   4.6.0                hddb2be6_3    conda-forge
libuv                     1.48.0               hcfcfb64_0    conda-forge
libwebp-base              1.4.0                hcfcfb64_0    conda-forge
libxcb                    1.16                 hcd874cb_0    conda-forge
libxml2                   2.12.7               h0f24e4e_4    conda-forge
libzlib                   1.3.1                h2466b09_1    conda-forge
m2w64-gcc-libgfortran     5.3.0                         6    conda-forge
m2w64-gcc-libs            5.3.0                         7    conda-forge
m2w64-gcc-libs-core       5.3.0                         7    conda-forge
m2w64-gmp                 6.1.0                         2    conda-forge
m2w64-libwinpthread-git   5.0.0.4634.697f757               2    conda-forge
markupsafe                2.1.5           py310h8d17308_0    conda-forge
matplotlib                3.9.1                    pypi_0    pypi
mkl                       2021.4.0                 pypi_0    pypi
mkl-devel                 2021.4.0           h57928b3_730    conda-forge
mkl-include               2021.4.0           h0e2418a_729    conda-forge
mpmath                    1.3.0              pyhd8ed1ab_0    conda-forge
msys2-conda-epoch         20160418                      1    conda-forge
munch                     4.0.0              pyhd8ed1ab_0    conda-forge
networkx                  3.3                pyhd8ed1ab_1    conda-forge
numcodecs                 0.13.0                   pypi_0    pypi
numpy                     1.26.4          py310hf667824_0    conda-forge
opencv                    4.9.0           qt6_py310h36cc372_615    conda-forge
openexr                   3.2.2                h72640d8_1    conda-forge
openh264                  2.4.1                h63175ca_0    conda-forge
openjpeg                  2.5.2                h3d672ee_0    conda-forge
openssl                   3.3.1                h2466b09_2    conda-forge
packaging                 24.1                     pypi_0    pypi
pandas                    2.2.2           py310hb4db72f_1    conda-forge
pcre2                     10.43                h17e33f8_0    conda-forge
pillow                    10.3.0          py310h3e38d90_1    conda-forge
pip                       24.1.2             pyhd8ed1ab_0    conda-forge
pixman                    0.43.4               h63175ca_0    conda-forge
plyfile                   1.0.3              pyhd8ed1ab_0    conda-forge
pthread-stubs             0.4               hcd874cb_1001    conda-forge
pthreads-win32            2.9.1                hfa6e2cd_3    conda-forge
pugixml                   1.14                 h63175ca_0    conda-forge
py-opencv                 4.9.0           qt6_py310h6d38b3d_615    conda-forge
pycparser                 2.22               pyhd8ed1ab_0    conda-forge
pydantic                  2.8.2              pyhd8ed1ab_0    conda-forge
pydantic-core             2.20.1          py310hc226416_0    conda-forge
pynvml                    11.5.3                   pypi_0    pypi
pyopengl                  3.1.6              pyhd8ed1ab_1    conda-forge
pyparsing                 3.1.2                    pypi_0    pypi
pysocks                   1.7.1              pyh0701188_6    conda-forge
python                    3.10.14         h4de0772_0_cpython    conda-forge
python-dateutil           2.9.0              pyhd8ed1ab_0    conda-forge
python-tzdata             2024.1             pyhd8ed1ab_0    conda-forge
python_abi                3.10                    4_cp310    conda-forge
pytorch-cuda              11.8                 h24eeafa_5    pytorch
pytorch-mutex             1.0                        cuda    pytorch
pytz                      2024.1             pyhd8ed1ab_0    conda-forge
pyyaml                    6.0.1           py310h8d17308_1    conda-forge
qt6-main                  6.7.2                h527115d_2    conda-forge
requests                  2.31.0             pyhd8ed1ab_0    conda-forge
scikit-image              0.24.0                   pypi_0    pypi
scipy                     1.13.1          py310h46043a1_0    conda-forge
setuptools                71.0.4             pyhd8ed1ab_0    conda-forge
simple-knn                0.0.0                    pypi_0    pypi
six                       1.16.0             pyh6c4a22f_0    conda-forge
snappy                    1.2.1                h23299a8_0    conda-forge
svt-av1                   2.1.0                he0c23c2_0    conda-forge
sympy                     1.13.0             pyh04b8f61_3    conda-forge
tbb                       2021.11.0                pypi_0    pypi
tifffile                  2024.7.24                pypi_0    pypi
tk                        8.6.13               h5226925_1    conda-forge
torch                     2.3.1+cu121              pypi_0    pypi
torchaudio                2.3.1+cu121              pypi_0    pypi
torchvision               0.18.1+cu121             pypi_0    pypi
tqdm                      4.66.4             pyhd8ed1ab_0    conda-forge
typing-extensions         4.12.2               hd8ed1ab_0    conda-forge
typing_extensions         4.12.2             pyha770c72_0    conda-forge
tzdata                    2024a                h0c530f3_0    conda-forge
ucrt                      10.0.22621.0         h57928b3_0    conda-forge
urllib3                   2.2.2              pyhd8ed1ab_1    conda-forge
vc                        14.3                h8a93ad2_20    conda-forge
vc14_runtime              14.40.33810         ha82c5b3_20    conda-forge
vs2015_runtime            14.40.33810         h3bf8584_20    conda-forge
vs2019_win-64             19.27.29111          ha89c4d3_2    anaconda
vswhere                   2.8.4                haa95532_0    anaconda
wheel                     0.43.0                   pypi_0    pypi
win_inet_pton             1.1.0              pyhd8ed1ab_6    conda-forge
x264                      1!164.3095           h8ffe710_2    conda-forge
x265                      3.5                  h2d74725_3    conda-forge
xorg-libxau               1.0.11               hcd874cb_0    conda-forge
xorg-libxdmcp             1.1.3                hcd874cb_0    conda-forge
xz                        5.2.6                h8d14728_0    conda-forge
yaml                      0.2.5                h8ffe710_2    conda-forge
zlib                      1.3.1                h2466b09_1    conda-forge
zstandard                 0.23.0          py310he5e10e1_0    conda-forge
zstd                      1.5.6                h0ea2cb4_0    conda-forge

**Comment by Florian-Barthel** [comment]
It looks like everything is installed correctly. Are you getting any error message? 

Sometimes, if you specify a --data_path with many files (eg: "C:\"), it can take up quiet a while until you see the application, since it is searching for all ply files in C:\

**Comment by xy-gao** [comment]
same here on Windows11. Turns out that downgrade imgui_bundle to 1.3.0 will make it work. the viewer showed up.

**Comment by Florian-Barthel** [comment]
I use imgui-bundle 1.5.2 on win10 and it works. The only env difference I noticed is pyopengl 3.1.6. I use pyopengl 3.1.7. This probably does not change anything since its just a minor change, but it might be worth a try.
